### ISSUE TITLE
Issue label sync broke

### ISSUE THREAD

**Issue by benhylau** [hyphacoop/organizing#265]
<sup>_This initial comment is collaborative and open to modification by all._</sup>
## Task Summary

🎟️ **Re-ticketed from:** #146   
🗣 **Loomio:** N/A  
📅 **Due date:** N/A  
🎯 **Success criteria:** Fix it.  

Used to work -> https://github.com/hyphacoop/organizing/runs/514125997?check_suite_focus=true#step:3:33
Now doesn't -> https://github.com/hyphacoop/organizing/runs/560378965?check_suite_focus=true#step:3:36

**Comment by benhylau** [comment]
Failing for 3 new repos:

- hypha.coop
- meshmedia
- remote

Seeing a dependency had a [recent change](https://github.com/vanilla/github-sync/commit/b93a91523f812020c4fe88069bab4dcd1e66c336) pinned to [commit before the change](https://github.com/hyphacoop/organizing/actions/runs/79359772/workflow#L49) but [still fails](https://github.com/hyphacoop/organizing/runs/590612548?check_suite_focus=true#step:3:41).

Basically all the label creation POSTs are 404.

**Comment by benhylau** [comment]
@dcwalk @ASoTNetworks ^

**Comment by benhylau** [comment]
@ASoTNetworks yea it could be this [access token](https://github.com/hyphacoop/organizing/settings/secrets) as you said in chat, I was afraid to touch it bc I don't know what else @hyphacoop/infrastructure-wg uses it for and how it's stored. Maybe @hyphacoop/robots and some worker script uses it too? If it's reset would it automatically get updated in other places?

**Comment by ASoTNetworks** [comment]
That is the token I was wondering about too and didn't want to touch it.

**Comment by benhylau** [comment]
You can try searching possible places for reference and then reset it.

**Comment by YurkoWasHere** [comment]
APIKEY - Found that existing API KEY was stored in passbolt. API key was generated under a special user hyphacoop_bot.

- Removed GITHUB_API_SECRET to try to re-add it.
- Github would NOT let me re-add the secret variable with error `Failed to add secret. Name is invalid` 
- Very little info about this error online (possible indicating something new?!?)
- Short research indicated that `GITHUB_` MAY be a reserved  prefix
- Changed variable name to  `API_TOKEN_HYPHACOOP_BOT`
- Updated script to match

**Comment by YurkoWasHere** [comment]
@hyphacoop_bot was not part of some new repos 
(https://github.com/orgs/hyphacoop/teams/robots/repositories)

[See Note](https://github.com/hyphacoop/organizing/blob/master/.github/workflows/sync-labels.yml#L22)

Following repos added to group

- hypha.coop - old repo was archived new repo never added to group
- meshmedia - added
- remote - added
